src/components/Disputes/PrecedenceCampaign/PrecedenceCampaignBanner.js

```diff
@@ -0,0 +1,74 @@
+import React, { useCallback, useState } from 'react'
+import { Button, GU, useTheme } from '@aragon/ui'
+import Modal from './PrecedenceCampaignModal'
+
+const KEY_PREFIX = 'PC_DISPUTE_BANNER_READ'
+
+function PrecedenceCampaignBanner({ disputeId }) {
+  const theme = useTheme()
+  const [bannerVisible, setBannerVisible] = useState(
+    localStorage.getItem(`${KEY_PREFIX}${disputeId}`) === null
+  )
+  const [modalOpened, setModalOpened] = useState(false)
+
+  const closeBanner = useCallback(() => {
+    localStorage.setItem(`${KEY_PREFIX}${disputeId}`, 'true')
```

Comment:
Maybe be useful to have this as `${KEY_PREFIX}_${disputeId}`.

But it may also be useful to use a small API around this, that does serialization, like [`StoredList`](https://github.com/aragon/aragon/blob/master/src/StoredList.js).